src/data/balance/pokemon-evolutions.ts

```diff
@@ -1609,12 +1504,8 @@ export const pokemonEvolutions: PokemonEvolutions = {
       "mint-cream",
       1,
       EvolutionItem.STRAWBERRY_SWEET,
-      new SpeciesEvolutionCondition(
-        (_p) =>
-          globalScene.arena.biomeType === Biome.SEA
-          || globalScene.arena.biomeType === Biome.BEACH
-          || globalScene.arena.biomeType === Biome.LAKE
-          || globalScene.arena.biomeType === Biome.SEABED,
+      new SpeciesEvolutionCondition((_p) =>
+        globalScene.arena.isInBiome([Biome.MOUNTAIN, Biome.CAVE, Biome.ICE_CAVE, Biome.FAIRY_CAVE, Biome.SNOWY_FOREST]),
```

Comment:
```suggestion
        globalScene.arena.isInBiome([Biome.SEA, Biome.BEACH, Biome.LAKE, Biome.SEABED]),
```
copy/paste issue seems like